utilities/workers/worker_manager.py

```diff
@@ -3,6 +3,11 @@
 """
 
 import multiprocessing as mp
+import types
+
+from typing import Tuple
+from utilities.workers import worker_controller
+from utilities.workers import queue_proxy_wrapper
```

Comment:
take a look at this section of the python style guide for import formatting
https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2253226033/Python+Style+Convention#Imports